src/guides/bioinformatics/exporting-data-for-auspice.md

```diff
@@ -0,0 +1,149 @@
+
+Filters in the bottom of the page are specified by   `meta.filters: Array<string>`. The values should correspond to keys in `node_attrs`, and auspice traverses the tree to find the set of all values. If a colouring was provided for this key then Auspice will attempt to display the title rather than the key. Selecting filters here behaves the same was as sidebar data filtering (see [a2], above).
+
+---
+
+### Exporting data via Augur
+
+We now consider how information flows through Augur, specifically `augur export v2`  which produces the dataset (Auspice) JSON described above. Augur has two main types of “metadata”  which we’ll refer to as “TSV-data” (provided with the `--metadata` argument) and “node-data” which are JSONs provided via `—node-data`. 
+
+At a high level, we combine the provided TSV-data and node-data and then export a subset of this data. This includes certain special keys (e.g. `num_date`) as well as any keys you specify as colorings, filters or geo-resolutions. Typically, node-data information is automatically a coloring whereas TSV-data must be explicitly exported [3]. We’ll now dive into the specifics a bit more!
+
+This guide is based on Augur version 16.0.1. We will endeavour to keep this up-to-date however if you are using a newer version  it may pay to check the [changelog](https://github.com/nextstrain/augur/blob/master/CHANGES.md) for any salient changes.
+
+
+**How are metadata inputs combined?**
+
+Node-data JSONs are combined using a deep-merge approach, where we recurse into dictionaries. For non-dictionary values we will overwrite previously seen values, so the order of JSONs provided matters! The information for each node is then added to the per-node (per-strain) TSV-data, with the node-data being used in the case of duplicate keys. As a contrived example:
+
+```js
+// node-data file 1
+nodes: {NODE_NAME: {country: "USA", region: "North America"}}
+// node-data file 2
+nodes: {NODE_NAME: {country: "Canada"}}
+// metadata TSV 
+{NODE_NAME: {country: "Mexico", serotype: "ABC"}}
+// resulting merged metadata
+nodes: {NODE_NAME: {country: "Canada", region: "North America", serotype: "ABC"}}
+```
+
+Note that for backwards compatibility, some keys are changed as they are read. Currently `authors → author` and  `numdate → num_date`.
+
+**What are the colorings, filters and geo-resolutions?**
+
+Dataset colorings are created via the combination of:
+* All key-value pairs specified in node-data JSONs, unless they match a hardcoded-list of exclusions. This has the effect that data provided in node-data JSONs will form a colouring and thus be exported in the dataset JSON.
+* Command-line provided colorings
+* Colorings specified in the auspice-config JSON.
+* Special cases are automagically added if the provided data support them:
+    * `gt` if mutations are defined.
+    * `num_date` if dates are defined (on any nodes)
+    * `clade_membership` if `clade_membership` or `clade_annotation` are defined on any nodes.
```

Comment:
Learning from this [thread](https://discussion.nextstrain.org/t/how-to-specify-what-metadata-is-shown-in-sample-details/1201), coloring has an additional effect to specify what gets exported to tree JSON and then shown in [a9]. It's indicated in the first point but could be helpful to be more explicit about that connection. Thank you for this wonderful doc!